webservices/legal_docs/load_legal_docs.py

```diff
@@ -492,3 +492,76 @@ def remap_citations(archived_murs):
         # Include regulations and us_code citations in the re-map
         mur.citations = get_citations(map(lambda c: c['text'], list(mur.citations['regulations']) + list(mur.citations['us_code'])))
         yield mur
+
+def enable_stemming():
+    es = utils.get_elasticsearch_connection()
+    settings = {
+        "mappings": {
+            "_default_": {
+                "properties": {
+                    "no": {
+                        "type": "string",
+                        "index": "not_analyzed"
+                    },
+                    "text": {
+                        "type": "string",
+                        "analyzer": "english"
+                    },
+                    "name": {
+                        "type": "string",
+                        "analyzer": "english"
+                    },
+                    "description": {
+                        "type": "string",
+                        "analyzer": "english"
+                    },
+                    "summary": {
+                        "type": "string",
+                        "analyzer": "english"
+                    }
+                }
+            }
+        },
+        "settings": {
+            "analysis": {"analyzer": {"default": {"type": "english"}}}
+        }
+    }
+
+    docs_new = Index('docs_new', using=es)
+    if not docs_new.exists():
+        es.indices.create('docs_new', settings)
```

Comment:
I encountered issues with corruption when the settings were different. Sure docs_backup is a better name, thanks.